CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ In this release, Puppet 6 is officially supported and [support for Puppet 4](htt
 - Use types and remove validate [\#172](https://github.com/voxpupuli/puppet-firewalld/pull/172) ([jfroche](https://github.com/jfroche))
 - Ipset options [\#170](https://github.com/voxpupuli/puppet-firewalld/pull/170) ([jfroche](https://github.com/jfroche))
 - Add description and short option for firewalld zone [\#169](https://github.com/voxpupuli/puppet-firewalld/pull/169) ([jfroche](https://github.com/jfroche))
+- Add firewalld config options [\#168](https://github.com/voxpupuli/puppet-firewalld/pull/168) ([jfroche](https://github.com/jfroche))
```

Comment:
Interesting... Why did this suddenly appear?

Reply:
🤷‍♀